### ISSUE TITLE
Convoys don't behave correctly when navgrid stops short of the target

### ISSUE THREAD

**Issue by jaj22** [official-antistasi-community/A3-Antistasi#827]
*Version:* 2.2+

*Mods:* Worse on specific maps. Example on Tembelan.

*Environment*: Any

***Issue:***
Example, but there are probably plenty more like it:

> around every 10 minutes vehicle's would spawn, drive past HQ then pause at the end of the blue line marked on the picture, crew would then despawn leaving vehicle's behind or they would  disembark.

![Tembelan convoy pathing](https://user-images.githubusercontent.com/562040/75577401-d0ce6d80-5a59-11ea-8a0c-a3d524fd4b21.png)

This looks like the navgrid terminates too far from the resource or outpost. The ground convoy path-following code only terminates when the vehicle gets within 100m distance of the target, not when the route runs out of entries. 

In some cases the navgrids may be missing roads. In other cases there may simply be no suitable roads near a target. In the short term there's an easy workaround to simply drive towards the target location when the route arrays run out.

**Comment by jaj22** [comment]
Not what I thought it was. The convoy code is basically correct, but Arma gives up on finding a vehicle path to outpost_6 on Tembelan because it's completely surrounded by forest.

We will need a way to specify that there's no road-reinforcement path to specific markers.

**Comment by wurzel0701** [comment]
Done in the garrison rework, but I am not sure if it is worth it to try to implement that for the current version, it is a lot of work. I however should do something like the old killzones system to send only a specific amount of convoys to one location

**Comment by jaj22** [comment]
Yeah, I'll bail on this one until the garrison merge. Sounds like Tembelan is off the servers for the moment anyway, so hopefully it's not causing major problems on other maps.
